va-gov/pages/503.html

```diff
@@ -0,0 +1,56 @@
+---
+permalink: false
+private: true
+---
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta http-equiv="Content-type" content="text/html; charset=utf-8">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'none'; base-uri 'self'; connect-src 'self'; form-action 'self'; img-src 'self' data:; script-src 'self'; style-src 'unsafe-inline'">
+  <meta content="origin" name="referrer">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>VA.gov isn't working right now: VA.gov</title>
+  <style type="text/css" media="screen">
+    .container {
+      max-width: 700px;
+      margin: 1.5em auto;
+      color: #323a45;
+    }
+
+    h1, h2 {
+      font-family: "Bitter", "Georgia", "Cambria", "Times New Roman", "Times", serif;
+      line-height: 1.3;
+      margin-bottom: .25em;
+      margin-top: 1.5em;
+      font-size: 1.3rem;
+    }
+
+    h1 {
+      margin-top: 1em;
+      font-size: 1.6rem;
+    }
+
+    p {
+      font-family: "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif;
+      line-height: 1.5;
+      margin-bottom: 0.5em;
+      margin-top: 0.5em;
+    }
+
+  </style>
+</head>
+
+<body>
+  <div class="container">
+    <a href="/">
+      <img width="250" src="{% include "va-gov/includes/logo-350px-base64.html" %}" />
```

Comment:
@ncksllvn I'm pretty sure this will come through from the previous 503 work. Just adding here for posterity.

```suggestion
      <img width="250" alt="Go to VA.gov" src="{% include "va-gov/includes/logo-350px-base64.html" %}" />
```